ci/environment-3.6.yml

```diff
@@ -4,13 +4,11 @@ channels:
 dependencies:
   - flake8
   - ipywidgets
-  - dask=0.17.4
-  - distributed=1.21.8
+  - dask
+  - distributed
```

Comment:
I would prefer to keep these explicitly pinned if possible, just so it's clear what we're testing against.  I think that we should bump the versions to 0.19.2 and 1.23.2, the latest versions as of today.